.github/workflows/build-on-windows.yml

```diff
@@ -0,0 +1,286 @@
+
+      - name: 🔽 Install Visual Studio Components for C++, CMake, Win10SDK
+        shell: pwsh
+        run: |
+          $VSConfig = Resolve-Path ".\.github\.vsconfig" | select -ExpandProperty Path
+          cd "C:\Program Files (x86)\Microsoft Visual Studio\Installer\"
+          $InstallPath = "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise"
+          $Arguments = ('/c', "vs_installer.exe", 'modify', '--installPath', "$InstallPath", '--config', "$VSConfig", '--quiet', '--norestart', '--nocache')
+          $process = Start-Process -FilePath cmd.exe -ArgumentList $Arguments -Wait -PassThru -WindowStyle Hidden
+          $process.ExitCode
+
+      - name: 📥 Setup VC Environment (➔ vcvarsall)
+        run: call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" %PLATFORM%
+
+      - name: 🔽 Update VCPKG
+        run: |
+          cd C:\vcpkg
+          git pull --quiet
+          bootstrap-vcpkg.bat -disableMetrics
+          vcpkg integrate install --triplet=${{ matrix.config.VCPKG_TARGET_TRIPLET }} --feature-flags=manifests
+          vcpkg version
+
+      - name: 🎯 Cache VCPKG
+        id: cache-vcpkg
+        uses: actions/cache@v1 # https://github.com/actions/cache
+        with:
+          #path: ..\build\vcpkg_installed
+          path: ~\AppData\Local\vcpkg
+          key: cache-windows-vcpkg-${{ matrix.config.VCPKG_TARGET_TRIPLET }}-${{ github.ref }}-${{ github.run_number }}
+          restore-keys: |
+            cache-windows-vcpkg-${{ matrix.config.VCPKG_TARGET_TRIPLET }}-${{ github.ref }}
+            cache-windows-vcpkg-${{ matrix.config.VCPKG_TARGET_TRIPLET }}
+
+      - name: ❔ CHECK important folders, to see if everything is present (before building)
+        run: |
+          dir %VULKAN_SDK%\Bin
+          dir %VULKAN_SDK%\runtime\%PLATFORM%
+          dir %LocalAppData%\vcpkg
+
+      - name: ✏️ CMake ➔ Configure (including VCPKG ➔ Install Dependencies)
+        run: |
+          cmake -G "Visual Studio 16 2019" -A ${{ env.PLATFORM }}                           ^
+                -S ${{ github.workspace }} -B ../build                                      ^
+                -DCMAKE_BUILD_TYPE=%BUILD_TYPE%                                             ^
+                -DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake            ^
+                -DVCPKG_TARGET_TRIPLET=${{ matrix.config.VCPKG_TARGET_TRIPLET }}            ^
+                -DBUILD_SHARED_LIBS=${{ matrix.config.BUILD_SHARED_LIBS }}                  ^
+                -DTT_BUILD_PCH=${{ matrix.config.TT_BUILD_PCH }}                            ^
+                -DCMAKE_VERBOSE_MAKEFILE=ON
+
+      - name: 🙏 CMake ➔ Build
+        run: |
+          cd ..\build
+          cmake --build . --config %BUILD_TYPE% -j %NUMBER_OF_PROCESSORS%
+
+      - name: 📦 CMake ➔ Install
+        run: cmake --install ..\build --config %BUILD_TYPE% --prefix ..\install --strip --verbose
```

Comment:
Install into a subdirectory of the build directory. Already using a variable to the final name of the artifact before zipping. That way you do not have to rename the directory later.